lib/p2p/Pool.ts

```diff
@@ -290,6 +296,25 @@ class Pool extends EventEmitter {
     }
   }
 
+  /**
+   * Get node info
```

Comment:
This comment should line up with the method name, so something like "Gets a node's reputation score and whether it is banned."